docs/book/stacks-and-components/component-guide/orchestrators/vertex.md

```diff
@@ -287,10 +228,20 @@ pipeline_instance.run(
         cron_expression="*/5 * * * *"
     )
 )
+
+# Run a pipeline every hour
+# starting in one day from now and ending in three days from now
+pipeline_instance.run(
+    schedule=Schedule(
+        cron_expression="0 * * * *"
+        start_time=datetime.datetime.now() + datetime.timedelta(days=1),
```

Comment:
Can we talk about timezones? I feel like its a bit unclear which timezone is being picked up?

Reply:
I will explain timezones in the Cron expression + in the `start_time` and `end_time` fields.
(e.g. google doc states this

> To explicitly set a timezone to the cron tab, apply a prefix in the cron tab: "CRON_TZ=${IANA_TIME_ZONE}" or "TZ=${IANA_TIME_ZONE}". The ${IANA_TIME_ZONE} may only be a valid string from IANA time zone database. For example, "CRON_TZ=America/New_York 1 * * * *", or "TZ=America/New_York 1 * * * *".

)

Reply:
@htahir1 I pushed some details about time zones (see https://github.com/zenml-io/zenml/pull/2310/commits/11ef15db1fce1fdbd1f33a7275050958ff5d946c) . Is it OK for you ?